scripts/kubestellar-snapshot.sh

```diff
@@ -349,15 +365,30 @@ else
         postgresql_pod=$(kubectl --context $helm_context -n kubeflex-system get pod postgres-postgresql-0 2> /dev/null || true)
         if [ -z "$postgresql_pod" ]; then
             echoerr "postgres-postgresql-0 pod not found!"
+            postgresql_job=$(kubectl --context $helm_context -n kubeflex-system get pod --no-headers -o name | grep "install-postgres" | head -n 2> /dev/null || true)
+            if [ -n "$postgresql_job" ]; then
+                echo -e "Found at least one ${COLOR_INFO}postgresql${COLOR_NONE} installation job that did not complete: ${COLOR_INFO}$postgresql_job${COLOR_NONE}"
+                if kubectl --context $helm_context -n kubeflex-system logs $postgresql_job | grep "toomanyrequests" ; then
+                    echoerr "there may be an issue pulling the postgresql image from Docker Hub."
+                fi
+            fi
         else
             postgresql_status=$(kubectl --context $helm_context -n kubeflex-system get pod postgres-postgresql-0 -o jsonpath='{.status.phase}' 2> /dev/null || true)
             echo -n -e "- ${COLOR_INFO}postgres-postgresql-0${COLOR_NONE}: pod=${COLOR_INFO}postgres-postgresql-0${COLOR_NONE}, status="
             echostatus "$postgresql_status"
         fi
+        for crd in controlplanes.tenancy.kflex.kubestellar.org postcreatehooks.tenancy.kflex.kubestellar.org ; do
+            echo -n -e "- CRD ${COLOR_INFO}$crd${COLOR_NONE} "
+            if [ -n "$(kubectl --context $helm_context get crd $crd --no-headers -o name 2> /dev/null || true)" ] ; then
```

Comment:
Wouldn't it be simpler to do the following instead?

```suggestion
            if kubectl --context $helm_context get crd $crd &> /dev/null; then
```